### DISCUSSION TITLE
Portable desktop apps

### DISCUSSION THREAD

**Discussion by abose** [phcode-dev/phoenix#1820] [Ideas]
Distribute portable versions of the app in all supported platforms.

* For mac, the app is already portable, you can just download it from the latest release in https://github.com/phcode-dev/phoenix-desktop/releases , extract the app from dmg and use it.
* For Linux too, the app is portable. Download the zipped files that ends with `amd64_linux_bin-GLIBC-2.33.tar.gz`(for linux distros newer than ubuntu 22), and extract and follow the readme.
* For windows, we dont directly distribute a portable version as no one has asked one yet, but you should be able to install the app and then copy the full folder at "C:\Users\<your user name>\AppData\Local\Phoenix Code\" and zip it. That should be a portable version. 

From Discission: [Introducing Phoenix Code Desktop- Now on Windows, Mac & Linux](https://github.com/orgs/phcode-dev/discussions/1047#discussioncomment-10494163)

Please upvote this discussion if you need this feature.

**Comment by blogcommente** [comment]
The intuitive interface and export options make it a top [vn video editor](https://vlognoweditor.com/vn-video-editor-vs-blender-video-editor/)    choice for video editing on the go.

**Comment by chlow95** [comment]
I tride to use it as portable app on a usb key, I've just copied the directory on the key  ;  it seems to run ... but unable to open files or directory, don't open the file manager = unusable ! sorry
do you have an issue ??
Chris

**Reply by abose** [reply]
Which os is this?

**Reply by chlow95** [reply]
oh ... windows 11
